packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.scss

```diff
@@ -3,6 +3,11 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
+.iui-dialog-default.presentation-instance-filter-dialog-content-container {
+  height: calc(var(--iui-component-height-large) * 7);
```

Comment:
Is dialog resizing still working with `height` and `width` applied?

Reply:
It does, but I changed the implementation with the latest commit so this is no longer relevant.